app/views/admin/blacklisted_emails/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_for(@admin_blacklisted_email) do |f| %>
+  <% if @admin_blacklisted_email.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@admin_blacklisted_email.errors.count, "error") %> prohibited this admin_blacklisted_email from being saved:</h2>
```

Comment:
There is a different way to do this if we head to i18n....

http://guides.rubyonrails.org/i18n.html#pluralization